decidim-core/app/cells/decidim/author/show.erb

```diff
@@ -1,6 +1,6 @@
-<% data = has_tooltip? ? { tooltip: render(:profile_minicard).html_safe } : nil %>
-<span class="author" data-author>
-  <%= content_tag :span, class: "author__container#{" is-compact" if layout == :compact}", data: do %>
+<% tooltip = options.has_key?(:demo) ? { tooltip: render(:profile_minicard).html_safe } : nil %>
```

Comment:
```suggestion
<%# If the options hash has the demo key it means we are in the decidim-design engine, so it is not a real-world scenario with actual users %>
<% tooltip = options.has_key?(:demo) ? { tooltip: render(:profile_minicard).html_safe } : nil %>
```

Reply:
The reasoning here on why adding this comment is that I had to double check why it was checking it, and I forgot that in the decidim-design engine we talk about Demo. 
An alternative would be to rename this `demo` key to something like `has_remote_tooltip`

Reply:
I would add the comment . We need to know also that `profile_minicard` is also the tooltip